### DISCUSSION TITLE
How do I enable DevMode on my usally (policy?) locked chromebook?

### DISCUSSION THREAD

**Discussion by WalkTheEarth** [3kh0/ext-remover#1334] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi!

I would like to know how I can to enable devmode on my (guessing) policy locked chromebook? 
I am planning to unenroll and log into my own account, or better yet to use Pollen.
`board name dedede and on v130, lightspeed filter agent (if needed idk man)`

**Comment by akabutnicer** [comment]
ask this in TN (titanium network)

**Comment by HelloIloveGimkit2** [comment]
> Hi!
> 
> I would like to know how I can to enable devmode on my (guessing) policy locked chromebook? I am planning to unenroll and log into my own account, or better yet to use Pollen. `board name dedede and on v130, lightspeed filter agent (if needed idk man)`

well since your dedede and v130 (which works) [br1ck](https://br1ck.vercel.app) is your best friend

**Reply by HelloIloveGimkit2** [reply]
and cuz you wanna unenroll

**Comment by schoolexploitkid** [comment]
buy a suzyqable bc its a dedede, then do br1ck

**Reply by prjamming** [reply]
can you use a suzyq cable on hana?

**Reply by schoolexploitkid** [reply]
no, only dedede

**Comment by nathblade16** [comment, marked as answer]
my advice, don't unenroll or enable devmode, it will get you in more trouble than its worth and illegal

**Comment by ghost** [comment]
I tried it on my v108 chromebook and works
powerwash and connect to wifi until you see checking updates or determining device configuration and connect as fast and go to guest mode and disconnect and type rollback and unenrolled

**Reply by Zakzoric** [reply]
your chromebook can do sh1mmer, that makes you lucky
